src/test/java/net/imagej/ops/OpMatchingServiceTest.java

```diff
@@ -38,7 +38,21 @@
 
 import java.util.List;
 
+import net.imglib2.Dimensions;
```

Comment:
We need to be careful about depending on ImgLib2 classes at this layer. Otherwise, it will make splitting out the `scijava-ops` project much harder. (I know: I see `DoubleType` was already imported below... but we should fix that too.)